### PR TITLE
README.md updated, player button board implemented with hardcoded pla…

### PR DESCRIPTION
**Pull request by Tymoteuszauke**
…yer ships

### CODE REVIEW

File: client/src/main/java/com/blackship/battlesheep/AppClient.java [hunk 1]
```diff
@@ -13,7 +13,6 @@
  * @since 27.07.2017
  */
 public class AppClient  {
-
```
Comment (piotr921) [inline client/src/main/java/com/blackship/battlesheep/AppClient.java:16]: :+1:

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java [hunk 2]
```diff
@@ -51,21 +53,39 @@ public void handleCreateEnemyBoardActionButton(ActionEvent actionEvent) throws I
...
-            enemyGridPane.add(enemyMastPositions.get(i), twoDimensionalBoardPosition[X], twoDimensionalBoardPosition[Y]);
-        }
+        IntStream.range(0, BOARD_SIZE * BOARD_SIZE).forEach(data -> {
+            int[] twoDimensionalBoardPosition = PositionUtils.calculateFromOneDimensionalPosition(data, BOARD_SIZE);
+            enemyGridPane.add(enemyMastPositions.get(data), twoDimensionalBoardPosition[X], twoDimensionalBoardPosition[Y]);
+        });
+//        for (int i = 0; i < BOARD_SIZE * BOARD_SIZE; i++) {
+//            int[] twoDimensionalBoardPosition = PositionUtils.calculateFromOneDimensionalPosition(i, BOARD_SIZE);
+//            enemyGridPane.add(enemyMastPositions.get(i), twoDimensionalBoardPosition[X], twoDimensionalBoardPosition[Y]);
+//        }
```
Comment (Dzionek95) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java:76]: this comment is not good :(

**Review by Dzionek95 — APPROVED** [review]
provide some java docs in future

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java (hunk 2 above)
Comment (piotr921) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/BoardController.java:76]: Please do not leave commented code.

File: client/src/main/java/com/blackship/battlesheep/fx/controllers/SalvoHandler.java [hunk 3]
```diff
@@ -3,14 +3,16 @@
...
 import com.blackship.battlesheep.communication.network.packet.PacketFactory;
 import com.blackship.battlesheep.communication.packet.Packet;
-import com.blackship.battlesheep.communication.packet.PacketBoards;
 import com.blackship.battlesheep.communication.packet.PacketMove;
 import com.blackship.battlesheep.communication.packet.enums.PacketType;
 import com.blackship.battlesheep.utils.LogUtils;
 import org.slf4j.Logger;
 
 import java.io.IOException;
-import java.util.*;
```
Comment (piotr921) [inline client/src/main/java/com/blackship/battlesheep/fx/controllers/SalvoHandler.java:13]: :+1:

File: client/src/main/java/com/blackship/battlesheep/fx/utils/ButtonUtils.java [hunk 4]
```diff
@@ -37,7 +38,7 @@ public static Button createStyledButton(int value) {
     public static List<Button> generateButtons(int boardSize) {
         List<Button> buttons = new ArrayList<>();
         IntStream
-                .rangeClosed(1, boardSize * boardSize)
+                .rangeClosed(BOARD_STARTING_POSITION, boardSize * boardSize)
```
Comment (piotr921) [inline client/src/main/java/com/blackship/battlesheep/fx/utils/ButtonUtils.java:41]: :+1:

**Review by piotr921 — APPROVED** [review]

**Pull request merged**